clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -47,7 +48,8 @@ trait GvcfTests { self: BaseIntegrationSpec =>
       "--version",
       key.version.toString,
       "--metadata-location",
-      tmpMetadata.toString
+      tmpMetadata.toString,
+      if(forceUpdate)"--force-update" else ""
```

Comment:
Can you add a new integration test that tests upserting without the force? Covering:
1. Fresh upsert
2. Filling in missing keys in an existing record
3. Failing to overwrite existing keys with a new value

I don't trust the client unit tests.

Reply:
Agreed. Client tests seem sorely lacking.